Azure Monitor: Filter namespaces by resource group (#100325)
grafana/grafana: docs/sources/datasources/azure-monitor/template-variables/index.md

You can specify these Azure Monitor data source queries in the Variable edit view's **Query Type** field.

| Name                    | Description                                                                                                                                                                         |
| ----------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| **Subscriptions**       | Returns subscriptions.                                                                                                                                                              |
| **Resource Groups**     | Returns resource groups for a specified subscription. Supports multi-value.                                                                                                         |

> | **Namespaces**          | Returns metric namespaces for the specified subscription. If a resource group is supplied the namespaces returned will be filtered to those that are present in the resource group. |

Reviewer: ```suggestion
| **Namespaces**          | Retrieves the metric namespaces for the specified subscription. If a resource group is provided, only the namespaces within that group are returned. |
```
